app/src/main/plugins.js

```diff
@@ -55,7 +55,7 @@ class Plugins {
   }
 
   async upgrade() {
-    await this._runNpm('install');
+    await this._runNpm('install --registry=https://registry.npmjs.org');
```

Comment:
Note the signature of `_runNpm`.

Reply:
Oops, sorry. I'll fix it when I get to a computer

Reply:
3e1fbde